To: vim_dev@googlegroups.com
Subject: Patch 7.3.036
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.036
Problem:    Win32 GUI: When building without menus, the font for dialogs and
	    tab page headers also changes.
Solution:   Define USE_SYSMENU_FONT always. (Harig G.)
Files:	    src/gui_w32.c


*** ../vim-7.3.035/src/gui_w32.c	2010-10-23 14:02:48.000000000 +0200
--- src/gui_w32.c	2010-10-25 20:29:15.000000000 +0200
***************
*** 289,301 ****
  
  #ifdef FEAT_MENU
  static UINT	s_menu_id = 100;
  
  /*
   * Use the system font for dialogs and tear-off menus.  Remove this line to
   * use DLG_FONT_NAME.
   */
! # define USE_SYSMENU_FONT
! #endif
  
  #define VIM_NAME	"vim"
  #define VIM_CLASS	"Vim"
--- 289,301 ----
  
  #ifdef FEAT_MENU
  static UINT	s_menu_id = 100;
+ #endif
  
  /*
   * Use the system font for dialogs and tear-off menus.  Remove this line to
   * use DLG_FONT_NAME.
   */
! #define USE_SYSMENU_FONT
  
  #define VIM_NAME	"vim"
  #define VIM_CLASS	"Vim"
*** ../vim-7.3.035/src/version.c	2010-10-24 14:33:38.000000000 +0200
--- src/version.c	2010-10-27 12:14:39.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     36,
  /**/

-- 
ARTHUR:  Bloody peasant!
DENNIS:  Oh, what a give away.  Did you hear that, did you hear that, eh?
         That's what I'm on about -- did you see him repressing me, you saw it
         didn't you?
                                  The Quest for the Holy Grail (Monty Python)

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\        download, build and distribute -- http://www.A-A-P.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
